File: src/app/model/attachement.ts
export interface Attachement {
  cardId: number,
  type: string,
  data: string,
  lastModified: number,
  createdAt: number,
  createdBy: string,
  deletedAt: number,
  extendedData: {
    filesize: number,
    mimetype: string
    info: {
      dirname: string,
      basename: string,
      extension: string,
      filename: string
    }
  },
  id: number
}

File: lib/model/attachement.dart
import 'dart:core';
import 'dart:ffi';

class Attachement {
  late int cardId;
  late String type;
  late String data;
  late Int? lastModified;
  late int? createdAt;
  late String createdBy;
  int? deletedAt;
  late ExtendedData extendedData;
  late int id;
}

class ExtendedData {
  int? filesize;
  late String mimetype;
  late Info info;
}

class Info {
  late String dirname;
  late String basename;
  late String extension;
  late String filename;
}
